spec/factories/accept_privacy_policy.rb

```diff
@@ -1,5 +1,5 @@
 FactoryBot.define do
   factory :accept_privacy_policy do
-    accepted_policy_id { true }
+    accepted_policy_id { ApiClient.get_latest_privacy_policy.id }
```

Comment:
Will this not hit the actual API, or create a VCR file? Probably worth hard coding it in the factories

Reply:
Currently it is cached.

Reply:
Currently it is cached.

Reply:
Test suite _really_ shouldn't be hitting the API. I think considering this is a Factory and you're in control of the data its fine to hard code a value. If you do want something random you can use `SecureRandom.uuid` - that is the same structure as the v4 UUIDs used by Dynamics